src/envoy/auth/http_filter.cc

```diff
@@ -154,9 +156,12 @@ std::string JwtVerificationFilter::Verify(HeaderMap& headers) {
     // Invalid JWT
     return Auth::StatusToString(jwt.GetStatus());
   }
-  /*
-   * TODO: check exp claim
-   */
+
+  // Check "exp" claim.
+  auto unix_timestamp = std::chrono::seconds(std::time(NULL)).count();
```

Comment:
Use `system_clock` and http://en.cppreference.com/w/cpp/chrono/time_point/time_since_epoch

Reply:
done